packages/flutter/test/material/sliver_refresh_control_test.dart

```diff
@@ -0,0 +1,1666 @@
+          RefreshIndicatorMode.inactive,
+        );
+      },
+      variant: const TargetPlatformVariant(<TargetPlatform>{TargetPlatform.android}),
+    );
+
+    testWidgets('buildRefreshIndicator progress', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Builder(
+            builder: (BuildContext context) {
+              return const SliverRefreshIndicator().buildRefreshIndicator(
+                context,
+                RefreshIndicatorMode.drag,
+                10,
+                100,
+                10,
+              );
+            },
+          ),
+        ),
+      );
+      expect(tester.widget<RefreshProgressIndicator>(find.byType(RefreshProgressIndicator)).progress, 10.0 / 100.0);
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Builder(
+            builder: (BuildContext context) {
+              return const SliverRefreshIndicator().buildRefreshIndicator(
+                context,
+                RefreshIndicatorMode.drag,
+                26,
+                100,
+                10,
+              );
+            },
+          ),
+        ),
+      );
+      expect(tester.widget<RefreshProgressIndicator>(find.byType(RefreshProgressIndicator)).progress, 26.0 / 100.0);
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Builder(
+            builder: (BuildContext context) {
+              return const SliverRefreshIndicator().buildRefreshIndicator(
+                context,
+                RefreshIndicatorMode.drag,
+                100,
+                100,
+                10,
+              );
+            },
+          ),
+        ),
+      );
+      expect(tester.widget<RefreshProgressIndicator>(find.byType(RefreshProgressIndicator)).progress, 100.0 / 100.0);
+    });
+  }
+
+  group('UI tests long list', uiTestGroup);
+
+  // Test the internal state machine directly to make sure the UI aren't just
+  // correct by coincidence.
+  group('state machine test long list', stateMachineTestGroup);
+
+  // Retest everything and make sure that it still works when the whole list
+  // is smaller than the viewport size.
+  testListLength = 2;
+  group('UI tests short list', uiTestGroup);
+
+  // Test the internal state machine directly to make sure the UI aren't just
+  // correct by coincidence.
+  group('state machine test short list', stateMachineTestGroup);
+
+  testWidgets('Does not crash when paintExtent > remainingPaintExtent', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: CustomScrollView(
+            physics: const BouncingScrollPhysics(
+              parent: AlwaysScrollableScrollPhysics(),
+            ),
+            slivers: <Widget>[
+              const SliverRefreshIndicator(),
+              SliverList(
+                delegate: SliverChildBuilderDelegate(
+                  childCount: 20,
+                  (BuildContext context, int index) {
```

Comment:
This is causing one of the analysis failures the named parameter should come after the builder.